packages/cli/src/lib/diff/handlers.ts

```diff
@@ -281,7 +282,7 @@ export async function handleAllFiles(
   promptFunc: PromptFunc,
 ) {
   for (const file of files) {
-    const { path } = file;
+    const path = file.path.split(sep).join(posix.sep);
```

Comment:
Is there a posix.normalize :)

Reply:
Yet but it doesn't do this :p. It's more about converting `a//b` into `a/b` and that kinda stuff